website/slack.shtml

```diff
@@ -0,0 +1,23 @@
+<!--#set var="dj_page_title" value="Slack" --><!--#include virtual="/header.shtml" -->
+
+<p>
+    DOMjudge offers a Slack Workspace where developers and users can discuss
+    any questions, comments, ideas and other DOMjudge related things they want.
+</p>
+<p>
+    Our workspace can be found at
+    <a href="https://domjudge-org.slack.com/" target="_blank">https://domjudge-org.slack.com/</a>.
+    To join as a new member, you can use
+    <a href="/slack/join" target="_blank">this page to sign up</a>.
+</p>
+<p>
+    Once you have joined the Slack Workspace, feel free to ask any DOMjudge-related
+    questions you might have. Please do note that it may sometimes take a bit longer
+    than a few minutes to get a response.
+
+    Because Slack has a limit of 10.000 messages that it will keep on the free plan,
```

Comment:
Actually is that archiving GDPR compliant if we do it as a service?

Reply:
You could say that the ground is consent since users are free to use this, and informed on this page before they join.

Reply:
We might also pin a message in #general telling them that we do it?

Reply:
There's not only consent, but also right to be forgotten: https://gdpr-info.eu/art-17-gdpr/

Reply:
Would `delete from messages where user_id = <theuserid>` (and deleting from `users`) be enough for this, if we get a request to remove the logs? We then miss other people quoting the messages and/or highlighting the user...

Reply:
Does the GDPR even apply to us as long we don't have the stichting DOMjudge foundation yet?

Reply:
🤷‍♂️ IANAL, but I also don't need that long of a history.

Reply:
Personally, I see chat as something ephemeral similar to real world chat.